subsys/bluetooth/audio/vocs_client.c

```diff
@@ -230,6 +230,7 @@ static void vcs_client_write_vocs_cp_cb(struct bt_conn *conn, uint8_t err,
 {
 	int cb_err = err;
 	struct bt_vocs *inst = lookup_vocs_by_handle(conn, params->handle);
+	static bool retry = true;
```

Comment:
Global for all connections? Shouldn't this be in the vocs instance?

Reply:
Great catch! I had almost forgot I added multi-conn support :D